server/controllers/chartsController.js

```diff
@@ -249,3 +244,13 @@ function isAnyTargetIncluded(studyTotals) {
     .filter((site) => site !== TOTALS_STUDY)
     .some((site) => studyTotals[site]?.targetTotal !== undefined)
 }
+
+function isTargetValueOnAllSites(fieldLabelValueMap, valueLabel) {
+  return Object.values(
+    fieldLabelValueMap.find(({ label }) => label === valueLabel).targetValues
+  ).every((target) => !!target)
+}
+
+function isNoValueGroup(value) {
+  return value === ''
```

Comment:
can we extract the hard-coded `''` to a constant to make the code more self-documenting?

Reply:
[cbdb19e](https://github.com/AMP-SCZ/dpdash/pull/217/commits/cbdb19ee4098d9b8d76793cb0d383fe9ad4e86d7)